### DISCUSSION TITLE
django models edit error

### DISCUSSION THREAD

**Discussion by ALISHERpy** [community/community#65833] [Programming Help]
### Body

soo i try to edit objects of Costumuser model in ADMIN panel.when i edit all the field and Save it,nothing changed at all,
 fuuuuuu*k(english) suukkaaa(russian).
why ?
i didn't anyt changes about admin panel settings....
 

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by seyLu** [comment]
#### Hi @ALISHERpy 

#### If you have set up logging in your app, then you can just trace the error in the logs.

- You can read more:
  * [Python logging official docs](https://docs.python.org/3/library/logging.html) 
  * [Real Python - Logging](https://realpython.com/python-logging/)
